Pod/Classes/ResultType.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  ResultType.swift
+//  Pods
+//
+//  Created by Bueno McCartney on 12/7/16.
+//
+//
+
+import Foundation
+
+public protocol ResultType {
+    associatedtype Value
+    
+    init(_ value: Value?)
```

Comment:
I mean, I’m not totally against it, I just want to make sure we don’t have one team doing `NoResponseType` and another team doing `nil`. Anyone else have thoughts? @jgrandelli @ryang1428 @JohnBueno ?